tasks/queue-pdrs/schemas/config.json

```diff
@@ -13,6 +13,8 @@
     "collection": { "type": "object" },
     "provider": { "type": "object" },
     "queueUrl": { "type": "string" },
-    "parsePdrMessageTemplateUri": { "type": "string" }
+    "parsePdrMessageTemplateUri": { "type": "string" },
+    "state_machine": { "type": "string" },
```

Comment:
So should this still be in the schema with the changes you've made to the message adapter?

Reply:
No.  I will remove them.

Reply:
The message adapter is published.